renovate.json5

```diff
@@ -0,0 +1,51 @@
+{
+  "extends": [
+    "config:recommended",
+    ":pinAllExceptPeerDependencies",
+  ],
+
+  "timezone": "Europe/London",
+
+  "dependencyDashboardApproval": false,
+  "prHourlyLimit": 0,
+  "prConcurrentLimit": 20,
+  "automerge": false,
+  "rangeStrategy": "pin",
+
+  "labels": ["dependencies"],
+
+  // opt-in to pre-commit beta feature
+  // https://docs.renovatebot.com/modules/manager/pre-commit/#enabling
+  "pre-commit": {
+    "enabled": true
+  },
```

Comment:
This repo has pre-commit hooks (which EE doesn't) so let try out renovate's pre-commit manager

It is beta, so we have to extra-explicitly opt in to it